core/src/game.ts

```diff
@@ -70,14 +70,14 @@ export default class Game {
   }
 
   handleCreatedGameEvent(event: GameCreatedEvent) {
-    this.players = new Array(event.playersCount).fill(new Player());
+    this.players = [...new Array(event.playersCount)].map(() => new Player());
```

Comment:
why need this change~?

Reply:
I think the original version will populate each element of the array with the same object, not a new one